### PR TITLE
Update changelog.md

### PR DESCRIPTION
**Pull request by amyblais**
For v5.23.2 dot release

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.23 - [Quality Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
 
+- **v5.23.2, release day TBD**
+  - Fixing an issue where
```
[the remark this answers is not in the document]
Comment (amyblais) [reply source/administration/changelog.md]: @lieut-data / @sudheerDev Please let me know if you have a short description of the issue I can communicate to users

[the remark this answers is not in the document]
Comment (sudheerDev) [reply source/administration/changelog.md]: @amyblais The 3rd party library we are using deployed a new version which should fix it for customers. We are waiting for confirmation on the resolution but at the moment we don't need to make a dot release.

[the remark this answers is not in the document]
Comment (amyblais) [reply source/administration/changelog.md]: Ah I see. I'll close this for now until we have full details

**Comment by amyblais** [comment]
@sudheerDev / @lieut-data I've reopened this since yesterday's solution didn't work for customers. We should at least give an FYI for users about the issue

**Comment by Willyfrog** [comment]
> 1 - "Current workaround is to clear the cache and delete the AppData folder" --> how? 
Users can delete appropiate app data folder, which are:
- windows: `%APPDATA%/Mattermost
- mac: `~\Library\Application Support\Mattermost`
- linux depends on install method, but usually defaults to `~\.config\Mattermost`

File: source/administration/changelog.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.23 - [Quality Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
 
+- **v5.23.2, release day TBD**
+  - Investigating an issue where users are experiencing high CPU and memory usage. Current workaround is to clear the cache. Workaround for affected desktop app users also includes deleting the AppData folder (Windows: ``%APPDATA%/Mattermost``; Mac: ``~\Library\Application Support\Mattermost``; Linux: ``~\.config\Mattermost``).
```
Comment (sudheerDev) [inline source/administration/changelog.md:10]: > Investigating an issue where users are experiencing high CPU and memory usage after prolonged use of desktop or webapp. 

I think we should say `prolonged use` to be clear.

File: source/administration/changelog.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sudheerDev) [reply source/administration/changelog.md]: @amyblais So far we only have one report, i think we should still wait for few more hours for other customers to get back.

File: source/administration/changelog.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.23 - [Quality Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
 
+- **v5.23.2, release day TBD**
+  - Investigating an issue where users are experiencing high CPU and memory usage after prolonged use of desktop app or webapp. Current workaround is to clear the cache. Workaround for affected desktop app users also includes deleting the AppData folder (Windows: ``%APPDATA%/Mattermost``; Mac: ``~\Library\Application Support\Mattermost``; Linux: ``~\.config\Mattermost``).
```
Comment (jasonblais) [inline source/administration/changelog.md:10]:
```suggestion
  - Investigating an issue where users are experiencing high CPU and memory usage after a prolonged use of Mattermost on the Desktop Apps or the browser. Current workaround is to clear the cache. Workaround for affected desktop app users also includes deleting the AppData folder (Windows: ``%APPDATA%/Mattermost``; Mac: ``~\Library\Application Support\Mattermost``; Linux: ``~\.config\Mattermost``).
```

**Pull request closed without merging**